### DISCUSSION TITLE
Error in installing MOOSE.

### DISCUSSION THREAD

**Discussion by d5sneha** [idaholab/moose#28577] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

hello,
i am facing this error while installing MOOSE.
(base) snehad@ubuntu:~$ conda create -n moose moose-dev=2024.08.30 mpich
Channels:
 - https://conda.software.inl.gov/public
 - conda-forge
Platform: linux-64
Collecting package metadata (repodata.json): failed

UnavailableInvalidChannel: HTTP 403 FORBIDDEN for channel public <https://conda.software.inl.gov/public>

The channel is not accessible or is invalid.

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations.

**Comment by GiudGiud** [comment]
This is tied to our new use of cloudyflare

 @milljm

**Reply by milljm** [reply]
Works for me... Both while at home and at work. As well as from my cell phone.

**Reply by GiudGiud** [reply]
same for me. But this is the 3rd poster reporting this

**Reply by milljm** [reply]
This is probably due to their country of origin blocking _outbound_ traffic. Not us blocking them.

**Reply by GiudGiud** [reply]
from the other post, when they reach the website on a browser they get a "are you human" challenge. They can pass it in the browser. conda cant.

It can also be cloudflare treating other countries differently

**Reply by milljm** [reply, marked as answer]
Okay, yeah that was helpful. I am in contact with our Cloudflare team. I am asking that this "feature" be disabled.

**Reply by milljm** [reply]
I have been told this was fixed yesterday at 14:30 MST.
